MoltenVK/MoltenVK/GPUObjects/MVKDescriptorSet.mm

```diff
@@ -354,6 +354,7 @@ static void populateAuxBuffer(mvk::SPIRVToMSLConversionConfiguration& shaderConf
                                                const VkDescriptorSetLayoutCreateInfo* pCreateInfo) : MVKVulkanAPIDeviceObject(device) {
 
 	_isPushDescriptorLayout = mvkIsAnyFlagEnabled(pCreateInfo->flags, VK_DESCRIPTOR_SET_LAYOUT_CREATE_PUSH_DESCRIPTOR_BIT_KHR);
+	_canUseMetalArgumentBuffer = !_isPushDescriptorLayout;	// Push constants don't use argument buffers
```

Comment:
```suggestion
	_canUseMetalArgumentBuffer = !_isPushDescriptorLayout;	// Push constants/descriptors don't use argument buffers
```

Granted, push constants don't either, but this is also for `VK_KHR_push_descriptor`.

Reply:
Thanks. Good catch. Done.